internal/provider/resource_wlan.go

```diff
@@ -343,6 +383,18 @@ func resourceWLANSetResourceData(resp *unifi.WLAN, d *schema.ResourceData, meta
 	d.Set("wlan_band", resp.WLANBand)
 	d.Set("no2ghz_oui", resp.No2GhzOui)
 
+	if resp.MinrateNgEnabled {
```

Comment:
I think the way this code is written means that if data rate controls are disabled then the old value will persist in the state?